README.md

```diff
@@ -47,14 +47,16 @@ Values set in your shell will override those in the `.env` file.
 
 **Provision your Elastic**
 
-In the future we hope to have better ways of doing this. However, for now, you need to manually seed your schema. To do this, create an ElasticSearch api instance, and call the following once per **new elastic instance**:
+You need to manually seed the ElasticSearch schema when you're first setting up. To do this, create an ElasticSearch api instance, and use the following helper script once per **new elastic instance**:
 
-```python
-from elasticsearch import Elasticsearch
-from simon.utils.elastic import _seed_schema
+```
+python setup_es.py
+```
 
-es = Elasticsearch(ELASTIC_URL, basic_auth=(ELASTIC_USER, ELASTIC_PASSWORD))
-_seed_schema(es)
+If you find that you want to delete the existing ElasticSearch schema and start fresh, you can use the `--nuke` option:
+
+```
+python setup_es.py --nuke
```

Comment:
This is beautiful thank you. I can do this later but perhaps we can make convert this to just a general utility script.